opm/simulators/flow/Main.cpp

```diff
@@ -35,11 +35,26 @@
 #include <opm/simulators/utils/DamarisOutputModule.hpp>
 #endif
 
+#include <iostream>
+#include <fcntl.h>  // for open()
+#include <unistd.h> // for dup2(), close()
+
+#include <iostream>
+#include <fcntl.h>  // for open()
+#include <unistd.h> // for dup2(), close()
+
+#include <iostream>
+#include <fcntl.h>  // for open()
+#include <unistd.h> // for dup2(), close()
```

Comment:
These are a lot of c-headers. Please use C++ wherever possible.

Maybe we can use the logging system to do this somehow or postpone this to later?